packages/vda-web3-client/src/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const veridaContractWhiteList = [
+  '0x2862BC860f55D389bFBd1A37477651bc1642A20B', //DID
+  '0x5c5CA3376b2C82f0322DB9dEE0504D2565080865', //NameRegistry
+  '0x17dFd83eFDD2D0c430E2cA4b01d1df93cDa9960b', //DID
```

Comment:
Why is DID here twice?

Reply:
That's for test.
The first one was created long ago. Resolving that contract generates a complicated DIDDocument.
For fresh test, I deployed contract again and added address here. Can remove one.
Of course, we should upate this list on deployment.